### PR TITLE
Add rules to tournaments

### PR DESCRIPTION
**Pull request by ytolstyk**

### CODE REVIEW

File: spec/helpers/scores_helper_spec.rb [hunk 1]
```diff
@@ -0,0 +1,15 @@
+require 'rails_helper'
```
Comment (azizova-leyla) [inline spec/helpers/scores_helper_spec.rb]: Delete this file or add tests.

File: spec/controllers/scores_controller_spec.rb [hunk 2]
```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
```
Comment (azizova-leyla) [inline spec/controllers/scores_controller_spec.rb]: delete this file

File: spec/helpers/scores_helper_spec.rb (hunk 1 above)
Comment (ytolstyk) [inline spec/helpers/scores_helper_spec.rb]: removed

File: spec/controllers/scores_controller_spec.rb (hunk 2 above)
Comment (ytolstyk) [inline spec/controllers/scores_controller_spec.rb]: removed

**Pull request merged**